cdap/resource_streaming_program_run.go

```diff
@@ -244,7 +274,7 @@ func resourceStreamingProgramRunDelete(d *schema.ResourceData, m interface{}) er
 
 	addr := getProgramAddr(config, d)
 	runsAddr := urlJoin(addr, "/runs")
-	r, err := getRunByFauxID(config, runsAddr, d.Id())
+	r, err := getRunByID(config, runsAddr, d.Id())
```

Comment:
Why do we still need this call? Isn't the whole point of the change that we already know the run ID now?